src/flatc.cpp

```diff
@@ -124,6 +124,9 @@ static void Error(const std::string &err, bool usage, bool show_exe_name) {
       "  --gen-name-strings Generate type name functions for C++.\n"
       "  --escape-proto-ids Disable appending '_' in namespaces names.\n"
       "  --gen-object-api   Generate an additional object-based API\n"
+      "  --cpp-variant VAR  What C++ variant to generate code for:\n"
+      "                       c++0x - (default) uses push_back and () for constructors\n"
```

Comment:
According to https://gcc.gnu.org/gcc-4.8/cxx0x_status.html is the first gcc with full C++11 support, but I checked https://gcc.gnu.org/gcc-4.7/cxx0x_status.html and it seems it has all the goodies too.
Anyway I'll choose 4.8 for safeness.